### DISCUSSION TITLE
[DOUBT] Can we run all publicly accessible models of Roboflow Universe without api key locally ?

### DISCUSSION THREAD

**Discussion by Greatz08** [roboflow/supervision#1537] [General]
Can we download those models and run locally without api ? I have doubts regarding what we can run for free locally and what we cannot so please explain in detail

**Comment by LinasKo** [comment]
Hi @HakaishinShwet,

The API is required to use any of the roboflow model - we don't provide weights directly.

Some models are accessible without an API key. Those are listed here: https://inference.roboflow.com/quickstart/aliases/. As far as I'm aware, all models on universe require an API key.

If you wish to use a model commercially, you need at least a Starter Plan. Here's more regarding the licensing: https://roboflow.com/licensing

**Reply by Greatz08** [reply]
@LinasKo  ok thankyou very much explaining that.
